containers/ecr-viewer/src/app/api/services/helpers/extended.ts

```diff
@@ -0,0 +1,39 @@
+import { getDb } from "@/app/api/services/database";
+import {
+  NewExtendedECR,
+  NewPatientAddress,
+  NewECRLabs,
+  Extended,
+} from "@/app/api/services/types/extended";
+
+/**
+ * Creates an eICR object
+ * @param ecr - the NewExtendedECR to be persisted
+ * @returns the created eICR object
+ */
+export async function createExtendedEcr(ecr: NewExtendedECR): Promise<void> {
+  await getDb<Extended>().insertInto("ecr_data").values(ecr).execute();
+}
+
+/**
+ * Creates a patient_address object
+ * @param patient_address - the NewPatientAddress to be persisted
+ * @returns the created patient_address object
+ */
+export async function createAddress(
+  patient_address: NewPatientAddress,
+): Promise<void> {
+  await getDb<Extended>()
+    .insertInto("patient_address")
+    .values(patient_address)
+    .execute();
+}
+
+/**
+ * Creates an eCR Lab object
+ * @param lab - the NewECRLabs to be persisted
+ * @returns the created eCR Lab object
+ */
+export async function createLab(lab: NewECRLabs): Promise<void> {
+  await getDb<Extended>().insertInto("ecr_labs").values(lab).execute();
+}
```

Comment:
I wasn't able to find a place where these two functions were used.

Reply:
looks like we don't - I'll delete